gdextension/extension_api.json

```diff
@@ -231816,6 +232100,23 @@
 					"is_vararg": false,
 					"is_virtual": true
 				},
+				{
+					"name": "_reload_scripts",
```

Comment:
Strange that there are so many new changes/additions to this file. Is this changes from 4.2 -> 4.3?

Reply:
I think it's because this branch is not up-to-date with godot 4.3 and we have added a lot of cherry-picks to the dev branch, that's why those changes were added.